src/iframe/src/canvasAPI/pixel.js

```diff
@@ -0,0 +1,29 @@
+import colors from '../colors.js'
+
+const backgroundColor = 7
+
+const getPixel = ({ x, y, ctx }) => {
+  x = x % 128
+  y = y % 128
+
+  // pixel: [r, g, b, a]
+  let pixel = ctx.getImageData(x, y, 1, 1).data
+
+  // If pixel isn't opaque default
+  if (pixel[3] !== 255) return backgroundColor
+
+  let color = colors.lookup(pixel)
+  // If color wasn't recognized return default
+  if (typeof color === "undefined") {
```

Comment:
Can you check if color is null instead? e.g. `if (!color)`

Reply:
This didn't get fixed before the PR was merged. Want me to make another PR making the change?

Reply:
No worries, it was the tiniest of things, already done.